# Backgrounds

### Cryo-Emergent

**Requirement**: Terran

You were born on Earth in the 20th century, and at some point in your life bought “Cryo-Insurance”. Upon your death your brain was frozen. Centuries later your brain was downloaded into an EGO and you woke up in an unfamiliar body in a whole new world. You are constantly fascinated by new technologies and this new odd world.

Skill Proficiencies: History. Choose 1 from [Diplomacy, Charm, Insight].
Tool Proficiency: [Gaming Set, Musical Instrument]

**Old World History**: You have advantage on History checks when related to old world technology or history.

### Spacer

You are more comfortable in null gravity and in a confined metal cockpit than you are planetside.

Skill Proficiencies: Acrobatics. Choose 1 from [Engineering, Computers, Pilot].
Tool Proficiency: Choose 1 from [Navigator's Datapad, Engineering's Kit]

**Spacewalker**: You do not have to roll ability check to move through null gravity.

### Duelist

You have risen in the ranks of duelists, and have earned your respect among the elite praetorian.

Skill Proficiencies: Athletics. Choose 1 from [Acrobatics, Charm, Intimidation]
Tool Proficiency: Choose 1 from [Musical Instrument, Navigator's Datapad, Medkit]
Weapon Proficiency: Pulse Rapier, Pulse Whip

Equipment: A Pulse Rapier, fine clothing, and 1000 credits.

### Netrunner

Since childhood you've retreated to the comfortable world of cyberspace.

Skill Proficiencies: Computers. Choose 1 from [Stealth, Investigate, Engineering]
Tool Proficiency: Hacker's Kit, and Forgery Kit

**Stealthy Hacker**: You have advantage on Stealth checks when trying to hack a system unnoticed.

Equipment: A hackers datapad, 3 wireless spycams, and 1000 credits.

### Gangster
You are an experienced criminal with a history of breaking the law. You have spent a lot of time among other criminals and still have contacts within the criminal underworld. You're far closer than most people to the world of murder, theft, and violence that pervades the underbelly of civilization.

Skill Proficiencies: Stealth. Choose 1 from [Trickery, Engineering, Intimidation]
Tool Proficiencies: Thieve's Tools. Choose 1 from [Forgery Kit, Poisoner's Kit, Disguise Kit]

Criminal Contact: You have a reliable and trustworthy contact who acts as your liaison to a network of other criminals. You know how to get messages to and from your contact.

Equipment: Knife ring, 3 wireless spycams, and 1000 credits.

```
```

### Smuggler
You make your living sneaking illegal goods past Authority checkpoints. You're good with your tongue, and even better in a cockpit.

Skill Proficiencies: Pilot. Choose 1 from [Stealth, Charm, Diplomacy]
Tool Proficiencies: Navigator's Tools, Forgery Kit

**Skills of the Trade**. You know how to hide things on a ship so they cannot be found. You also know where to look when trying to find hidden contraband.

Equipment: A fake ID, a good luck charm, and 1000 credits

### Detective

You've had a successful career in law enforcement. You've decided to go freelance since the private sector tends to pay better.

Skill Proficiencies: Investigation. Choose 1 from [Insight, Perception, History]
Tool Proficiencies: Choose 1 from [Disguise Kit, Engineering's Tools, Medkit]

**Lie Detector Kit** This kit includes an injector and 10 vials. When injected, you are more sensitive to the body language, and verbal tells of those you speak with and are better at detecting lies. For 10 minutes you have advantage on Insight checks when trying to discern if someone is lying. 

Equipment: Lie Detector Kit, a fedora, a scorcher, and 1000 credits.

### Marine

You've spent some time in the military. You have been trained by the best of the best, and are a war hardened veteran.

Skill Proficiencies: Athletics. Choose 1 from [Survival, Intimidation, Medic]
Tool Proficiencies: Medkit
Weapon Proficiencies: Rail Rifle
Armor Proficiencies: flak armor

**Veteran** As a veteran you can use a Charm check to attempt to get a discount when hiring Mercenaries. You are better at using the marine lingo, and as a result, any marines under your command gain a +1 to Armor Class and attack rolls.

Equipment: An old set of flak army (military issue), a medal awarded to you for your service, and 1000 credits.

### Lancer
You come from one of the elite houses that owns a large corporation, or you have been honored as a lancer of one of these elite houses. You maintain the rank and privileges of Lancer. As a member or lancer of an elite house you always have your house in the best interest.

Skill Proficiencies: Diplomacy. Choose 1 from [Charm, History , Medic]
Tool Proficiencies: Choose 1 from [Botanist's Kit, Musical Instrument, Disguise Kit]
Weapon Proficiencies: Pulse Rapier/Pulse Whip

**House Resources**: Depending on your social standing with your house, you might be able to persuade (Charm check) them to get you out of sticky situations, as well as provide you with resources. Be careful not to abuse this privilege, as the greedy pretorian leaders will likely ask for something in return.

Equipment: A badge marking your rank and affiliation, a Pulse Rapier, and 1500 credits.

\page

### Pilot

You have been trained to fly any kind of spacecraft. Your expertise makes you a valuable asset to any spacefaring venture.

Skill Proficiencies: Pilot. Choose 1 from [Engineering, Perception, Computers]
Tool Proficiencies: Navigator's Tools

If you already have the Pilot skill, you gain expertise in the Pilot skill.

**Frame Jack**: You have been surgically modified with hardware that allows you to jack into a ship's navigation systems. When frame jacked in you always know the status of ship systems and can simultaneously control navigation and weapon systems on smaller ships.

Equipment: A flight jacket you earned in flight school, a valid pilot's license and 1000 credits.

### Viral Star
You've become famous on the net for something. Whether it's your singing, acting, social commentator, or just a famous praetorian whose internet videos went viral. You tend to livestream your shenanigans, and take advertisement money. Like most internet celebrities, you might have a secret alter-ego.

Skill Proficiencies: Performance. Choose 1 from [Charm, Diplomacy, Insight]
Tool Proficiencies: Choose 1 from [Musical Instrument, Gaming Set, Disguise Kit]

**Universal Influence**: It's easy to get your voice out. Rallying supporters via the internet comes naturally to you. When you aren't incognito, there are perks to being famous, such as free drinks and lodging. These perks rarely come without strings, however.

Equipment: A camera drone programmed to follow and record your actions, or streams video live on SIGNET, a costume, and 1000 credits.

### Scientist

You have spent most of your life being educated, and now you are going out there to make a difference. Whether that is to discover, invent, or cure, you want to change the universe for better or for worse. You might be a researcher, doctor, inventor, chemist, physicist, or biologist.

Skill Proficiencies: Arcana. Choose 2 from [Nature, Medic, Investigation, History]
Tool Proficiencies: Choose 1 from [Botanist's Kit, MedKit, Poisoner's Kit]

**Field of Study**: You specialize in a specific field. You have advantage when rolling Science checks relating directly to that field of study.

Equipment: A molecular anylizer attachment for your datapad, a lab safety kit containing a lab coat, safety goggles, and gloves, and 1000 credits

### Engineer
You know the ins and outs of most ships and devices.

Skill Proficiencies: Engineering. Choose 1 from [Computers, Investigate, Pilot]
Tool Proficiencies: Engineering's Tools

**Choose 1:**

**Inventor**: You've invented something that is a component of a commonly used technology (And you love to  mention that fact). Because of this, you know that piece of technology exceptionally well. Gain advantage when Engineeringing with that specific technology. For example you might have invented the intake flux regulator on ship propulsion systems. You would have advantage on fixing or operating such systems.

**Improvised Engineerin**: You can macgyver tools together for any situation using the objects available to you. Gain advantage on Engineering checks when creating a tool for a specific situation.

Equipment: A micro 3D printer able to print simple objects up to 1 foot squared, a set of tinkering tools, and 1000 credits.

### Capitalist
You've gone from rags to riches, or at least that is the path you have set up for yourself. You've come a long way from your humble upbringing, and you are ambitious to rise to the top.

Skill Proficiencies: Diplomacy. Choose 1 from [Insight, History, Charm]

**Entrepreneur**: You are skilled at making business deals. You do all your business through a small corporation you chose a name for. You have 3 employees that run this small business for you. The business does not turn a profit unless you invest time and money into a business venture. The business can be any type of legal small business. The business venture can be anything legal or illegal. You must fund the venture directly by investing credits and making the appropriate business connections. The success of the venture is determined by a number of appropriate ability checks determined by the DM. Results can range from capital gains/losses, earned/ruined business relationships, to even legal trouble, or disputes with elite houses/corporations. Once you make the proposal and appropriate connections, the length of the business venture, the difficulty, and potential outcomes will be determined by the DM.

Equipment: A case of business cards, fine clothing, and 3000 credits
